python/oneflow/framework/docstr/tensor.py

```diff
@@ -1525,6 +1525,21 @@
     """,
 )
 
+add_docstr(
+    oneflow.Tensor.reshape_as,
+    """
+    Returns this tensor as the same shape as other.
+    self.reshape_as(other) is equivalent to self.reshape(other.sizes()).
+    This method returns a view if other.sizes() is compatible with the current shape.
+    See oneflow.Tensor.view() on when it is possible to return a view.
```

Comment:
```suggestion
    See :func:`oneflow.Tensor.view` on when it is possible to return a view.
```

改成这样应该有超链接效果

https://github.com/Oneflow-Inc/OneTeam/blob/master/tutorial/howto_write_api_docs.md#%E4%BA%A4%E5%8F%89%E5%BC%95%E7%94%A8